src/misc/motifs.c

```diff
@@ -479,9 +486,28 @@ int igraph_motifs_randesu_estimate(const igraph_t *graph, igraph_integer_t *est,
     long int sam;
     long int i;
 
+    if (no_of_nodes == 0) {
+        *est = 0;
+        return IGRAPH_SUCCESS;
+    }
```

Comment:
Let's move this below the following checks, so that these checks would be performed consistently for graphs of any size (even 0).